src/js/hooks/cycleCount/useCountStep.jsx

```diff
@@ -564,6 +565,23 @@ const useCountStep = () => {
     resetFocus();
   };
 
+  const applyImportFile = async (e) => {
+    if (e instanceof File) {
+      setImportFile(e);
+    }
+  };
+
+  const importItems = async () => {
+    try {
+      show();
+      const response = await cycleCountApi.importCycleCountItems(importFile, currentLocation?.id);
+      console.log(response);
```

Comment:
Is it by accident?

Reply:
No, I left it here so that I don't have an eslint error that `response` is not used. Anyway this will be adjusted in the next ticket, so I guess nothing wrong with leaving it for now.